app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddSidewalkSurfaceForm.kt

```diff
@@ -18,6 +19,10 @@ class AddSidewalkSurfaceForm : AStreetSideSelectForm<Surface, SidewalkSurfaceAns
     private var leftNote: String? = null
     private var rightNote: String? = null
 
+    override val otherAnswers = listOf(
+        AnswerItem(R.string.quest_sidewalk_answer_different) { applyAnswer(SidewalkIsDifferent) }
```

Comment:
For other answers that tag something, we always ask the user if he is sure **if I remember correctly**, so it would be a consistent UX to do that. See e.g. the AddSmoothnessForm, where you also got the option to say that it is a different surface.

Reply:
> For other answers that tag something, we always ask the user if he is sure **if I remember correctly**

It seems to be dependent on the quest, some change it without extra confirming, for example:
https://github.com/streetcomplete/StreetComplete/blob/d59747374351de6e4328a0c0f5f0a334e678b6e3/app/src/main/java/de/westnordost/streetcomplete/quests/bench_backrest/AddBenchBackrestForm.kt#L17-L19

or `AddBusStopShelterForm`, `TactilePavingCrosswalkForm`, `WayLitForm`, `AddReligionForm`, ...

As it is not consistent and we already have both situations in different quest; in this specific case it seemed (to me) better choice to not to bother the user with (in my opinion needless) extra confirmation, especially as the following quest will immediately ask for new sidewalk situation. 

But if you think it would be better to ask for confirmation anyway in this case, let me know and I'll implement it.

Reply:
I see, if it is not consistent, then I shall not force you to chase this.